message.go

```diff
@@ -23,6 +22,9 @@ type Message struct {
 	Time time.Time
 
 	// Compression codec used to encode the message value
+	//
+	// Deprecated: this field is ignored.  If compression is desired, then
+	// configure it on the Writer.
```

Comment:
I wonder if we should just remove the field. We'd be breaking programs at compile time, which is sad but could be better than silently changing the behavior at runtime?

Reply:
Removed!